container_templates/python/container_example/Dockerfile

```diff
@@ -0,0 +1,28 @@
+#Dockerfile is used to create an image
+
+#Set the base image.
+FROM python:3.9-slim
+
+RUN apt-get update
+RUN apt-get install -y gcc libpq-dev
```

Comment:
```suggestion
```

I don't think they'll need gcc, or if so they can add it. I'm not sure what `libpq` is? But, basically I think these should be very minimal to keep them small and let our downstream users make them bigger with extra dependencies if need be.